src/Model/Order.php

```diff
@@ -165,6 +166,13 @@ public function setTags(array $tags): self
         return $this;
     }
 
+    public function setAttributes(array $attributes): self
+    {
+        $this->attributes = $attributes;
```

Comment:
I wonder if there's an validation that we could/should do here 🤔 
I guess the value can be anything really - could be further arrays, ints, strings. I guess the keys need to be strings though, so it must be an associative array. I don't think numeric attributes would make much sense on our end (although I guess it could work, it would be confusing to say the least).
Wdyt?

Reply:
Have added some validation to ensure we receive string keys and that values are not considered "empty" 👍🏻